internal/db/migrations/postgres/0/100_server_tags_migrations.up.sql

```diff
@@ -0,0 +1,101 @@
+begin;
+
+-- This series of expressions fixes the primary key on the server table
+alter table session
+  drop constraint session_server_id_server_type_fkey;
+alter table server
+  drop constraint server_pkey;
+alter table server
+  drop column name;
+alter table server
+  add primary key (private_id);
+alter table server
+  add constraint server_id_must_not_be_empty
+  check(length(trim(private_id)) > 0);
+alter table session
+  add constraint session_server_id_fkey
+  foreign key (server_id)
+  references server(private_id)
+  on delete set null
+  on update cascade;
+
+-- Add the worker filter to the target_tcp table and session table
+alter table target_tcp
+  add column worker_filter text;
+alter table session
+  add column worker_filter text;
```

Comment:
Are there any restrictions on the `worker_filter` columns (min/max length, illegal characters, etc)?

Reply:
No. I'm not sure if there should be or not.